docs/apis/ledgerstate.md

```diff
@@ -0,0 +1,1112 @@
+* [/ledgerstate/branches/:branchID/children](#ledgerstatebranchesbranchidchildren)
+* [/ledgerstate/branches/:branchID/conflicts](#ledgerstatebranchesbranchidconflicts)
+* [/ledgerstate/outputs/:outputID](#ledgerstateoutputsoutputid)
+* [/ledgerstate/outputs/:outputID/consumers](#ledgerstateoutputsoutputidconsumers)
+* [/ledgerstate/outputs/:outputID/metadata](#ledgerstateoutputsoutputidmetadata)
+* [/ledgerstate/transactions/:transactionID](#ledgerstatetransactionstransactionid)
+* [/ledgerstate/transactions/:transactionID/metadata](#ledgerstatetransactionstransactionidmetadata)
+* [/ledgerstate/transactions/:transactionID/inclusionState](#ledgerstatetransactionstransactionidinclusionstate)
+* [/ledgerstate/transactions/:transactionID/consensus](#ledgerstatetransactionstransactionidconsensus)
+* [/ledgerstate/transactions/:transactionID/attachments](#ledgerstatetransactionstransactionidattachments)
+* [/ledgerstate/transactions](#ledgerstatetransactions)
+* [/ledgerstate/addresses/unspentOutputs](#ledgerstateaddressesunspentoutputs)
+
+
+Client lib APIs:
+* [client lib - `GetAddressOutputs()](#client-lib---getaddressoutputs)
+* [client lib - `GetAddressUnspentOutputs()](#client-lib---getaddressunspentoutputs)
+* [client lib - `GetBranch()](#client-lib---getbranch)
+* [client lib - `GetBranchChildren()](#client-lib---getbranchchildren)
+* [client lib - `GetBranchConflicts()](#client-lib---getbranchconflicts)
+* [client lib - `GetOutput()](#client-lib---getoutput)
+* [client lib - `GetOutputConsumers()](#client-lib---getoutputconsumers)
+* [client lib - `GetOutputMetadata()](#client-lib---getoutputmetadata)
+* [client lib - `GetTransaction()](#client-lib---gettransaction)
+* [client lib - `GetTransactionMetadata()](#client-lib---gettransactionmetadata)
+* [client lib - `GetTransactionInclusionState()](#client-lib---gettransactioninclusionstate)
+* [client lib - `GetTransactionConsensusMetadata()](#client-lib---gettransactionconsensusmetadata)
+* [client lib - `GetTransactionAttachments()](#client-lib---gettransactionattachments)
+* [client lib - `PostTransaction()](#client-lib---posttransaction)
+* [client lib - `PostAddressUnspentOutputs()](#client-lib---postaddressunspentoutputs)
+
+## `/ledgerstate/addresses/:address`
+Get an address details for a given base58 encoded address ID, such as output types and balances. For the client library API call balances will not be directly available as values because they are stored as a raw message. Balance cann be read after retrieving `ledgerstate.Output` instance, as presented in the examples.
+
+### Parameters
+| **Parameter**            | `addressID`      |
+|--------------------------|----------------|
+| **Required or Optional** | required       |
+| **Description**          | The address ID encoded in base58. |
+| **Type**                 | string         |
+
+### Examples
+
+#### cURL
+
+```shell
+curl http://localhost:8080/ledgerstate/addresses/:address \
```

Comment:
I would keep :address, but state in the line thereafter
where `:address` is the address, e.g. 6PQqFcwarCVbEMxWFeAqj7YswK842dMtf84qGyKqVH7s1kK 
because it is useful to highlight that the string that goes there is the address. See also my comment on `:branchID`